static/app/views/teamInsights/teamMisery.tsx

```diff
@@ -138,7 +140,8 @@ function TeamMiseryWrapper({
   const commonEventView = {
     id: undefined,
     query: 'transaction.duration:<15m team_key_transaction:true',
-    projects: projects.map(project => Number(project.id)),
+    // Empty array selects all projects for a user
+    projects: projects.length > 0 ? projects.map(project => Number(project.id)) : [-2],
```

Comment:
Why is there `[-2]`?

Reply:
-1 is all projects. Trying to get it to not query all projects. Maybe i should just return without making the queries at all

Reply:
returned early